tests/prague/test_state_transition.py

```diff
@@ -101,27 +101,35 @@ def test_general_state_tests(test_case: Dict) -> None:
     run_prague_blockchain_st_tests(test_case)
 
 
-# Run execution-spec-generated-tests for EIP-7002
-test_dir = "tests/fixtures/latest_fork_tests/fixtures/blockchain_tests/prague/eip7002_el_triggerable_withdrawals"
-
-
-@pytest.mark.parametrize(
-    "test_case",
-    fetch_prague_tests(test_dir),
-    ids=idfn,
+# Run temporary test fixtures for Prague
+test_dirs = (
+    "tests/fixtures/latest_fork_tests/fixtures/blockchain_tests/prague/eip7002_el_triggerable_withdrawals",
+    "tests/fixtures/latest_fork_tests/fixtures/blockchain_tests/prague/eip6110_deposits/deposits",
+    "tests/fixtures/latest_fork_tests/fixtures/blockchain_tests/prague/eip2537_bls_12_381_precompiles/bls12_g1add",
+    "tests/fixtures/latest_fork_tests/fixtures/blockchain_tests/prague/eip2537_bls_12_381_precompiles/bls12_g1mul",
+    "tests/fixtures/latest_fork_tests/fixtures/blockchain_tests/prague/eip2537_bls_12_381_precompiles/bls12_g2add",
+    "tests/fixtures/latest_fork_tests/fixtures/blockchain_tests/prague/eip2537_bls_12_381_precompiles/bls12_g2mul",
+    "tests/fixtures/latest_fork_tests/fixtures/blockchain_tests/prague/eip2537_bls_12_381_precompiles/bls12_pairing",
+    "tests/fixtures/latest_fork_tests/fixtures/blockchain_tests/prague/eip2537_bls_12_381_precompiles/bls12_g1msm",
+    "tests/fixtures/latest_fork_tests/fixtures/blockchain_tests/prague/eip2537_bls_12_381_precompiles/bls12_g2msm",
+    "tests/fixtures/latest_fork_tests/fixtures/blockchain_tests/prague/eip2537_bls_12_381_precompiles/bls12_map_fp_to_g1",
+    "tests/fixtures/latest_fork_tests/fixtures/blockchain_tests/prague/eip2537_bls_12_381_precompiles/bls12_map_fp2_to_g2",
 )
-def test_execution_specs_generated_tests_7002(test_case: Dict) -> None:
-    run_prague_blockchain_st_tests(test_case)
 
 
-# Run execution-spec-generated-tests for EIP-6110
-test_dir = "tests/fixtures/latest_fork_tests/fixtures/blockchain_tests/prague/eip6110_deposits/deposits"
+def fetch_temporary_tests(test_dirs: Tuple[str, ...]) -> Generator:
```

Comment:
Is this something we need to come back to later? Should we open an issue?

Reply:
This is to include only the tests which have the corresponding EIPs already implemented. Yes, this will go away as all the EIPs in the fork are implemented. This is already on the radar for post-prague re-factoring but I will create an issue.